niap-cc/Permissions/Tester/app/src/main/java/com/android/certifications/niap/permissions/SignaturePermissionTester.java

```diff
@@ -3180,26 +3323,36 @@ public Object invoke(Object o, Method method, Object[] objects)
                         throw new UnexpectedPermissionTestFailureException(e);
                     }
                     try {
-                        if (mDeviceApiLevel <= Build.VERSION_CODES.S_V2) {
+                        if(mDeviceApiLevel >= VERSION_CODES.UPSIDE_DOWN_CAKE){
+                            mTransacts.invokeTransact(Context.CAMERA_SERVICE,
+                                    Transacts.CAMERA_DESCRIPTOR,
+                                    Transacts.injectCamera, mContext.getPackageName(),
+                                    "",
+                                    "", injectionCallback);
+                        } else if(mDeviceApiLevel >= VERSION_CODES.S_V2){
                             mTransacts.invokeTransact(Transacts.CAMERA_SERVICE,
                                     Transacts.CAMERA_DESCRIPTOR,
                                     Transacts.injectCamera, mContext.getPackageName(),
                                     "test-internal-cam",
-                                    "test-external-cam", injectionCallback, injectionSession);
+                                    "test-external-cam", injectionCallback);
```

Comment:
Let me update the conditional expression for this version check. there was a mistake.
I updated this routine for prvious update.
The parameter was removed as of android t, and as I see the source code for android T, the new parameters were applied only android t. 
So we should change >=to > in this case. But for clarity I would like to use '==' operator instead of '>'.

Reply:
Thanks for the additional details!